xknx/devices/expose_sensor.py

```diff
@@ -30,8 +30,8 @@ def __init__(self,
         super(ExposeSensor, self).__init__(xknx, name, device_updated_cb)
 
         self.sensor_value = None
-        if value_type == "percent":
-            self.sensor_value = RemoteValueScaling(
+        if value_type == "binary":
+            self.sensor_value = RemoteValueSwitch(
```

Comment:
percent is treated like every other sensor now; binary is extra because of DPTBinary values.